tests/modules/asset_groups/test_models.py

```diff
@@ -95,11 +95,23 @@ def test_asset_group_sightings_jobs(flask_app, db, admin_user, test_root, reques
     assert not ps['detection']['inProgress']
     assert not ps['detection']['failed']
 
+    curation_progress = {
+        'skipped': False,
+        'start': ags1.get_curation_start_time(),
+        'end': None,
+        'inProgress': True,
+        'complete': False,
+        'failed': False,
+        'progress': 0.5,
+    }
+    assert ps['curation'] == curation_progress
+
```

Comment:
@Emily-Ke does that look good to you for an in-progress AGS? (the start field is a standard-formatted ISO string there)

Reply:
Yes